iocage/cli/activate.py

```diff
@@ -23,20 +23,40 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """activate module for the cli."""
 import click
+import libzfs
 
-import iocage.lib.ioc_common as ioc_common
-import iocage.lib.iocage as ioc
+import iocage.lib.Datasets
+import iocage.lib.Logger
```

Comment:
Tracing the file system access showed that files were included multiple times if using the from-syntax. I have now changed this, but will keep an eye on the import nesting issues.